source/MonoGame.Extended.Input/KeyboardStateExtended.cs

```diff
@@ -15,15 +15,37 @@ public KeyboardStateExtended(KeyboardState currentKeyboardState, KeyboardState p
             _previousKeyboardState = previousKeyboardState;
         }
 
-        public bool CapsLock => _currentKeyboardState.CapsLock;
-        public bool NumLock => _currentKeyboardState.NumLock;
+        public bool CapsLock
+        {
+#if FNA
+            get { return _currentKeyboardState.IsKeyDown(Keys.CapsLock); }
+#else
+            get { return _currentKeyboardState.CapsLock; }
+#endif
+        }
+        public bool NumLock
+        {
+#if FNA
+            get { return _currentKeyboardState.IsKeyDown(Keys.NumLock); }
+#else
+            get { return _currentKeyboardState.CapsLock; }
+#endif
+        }
         public bool IsShiftDown() => _currentKeyboardState.IsKeyDown(Keys.LeftShift) || _currentKeyboardState.IsKeyDown(Keys.RightShift);
         public bool IsControlDown() => _currentKeyboardState.IsKeyDown(Keys.LeftControl) || _currentKeyboardState.IsKeyDown(Keys.RightControl);
         public bool IsAltDown() => _currentKeyboardState.IsKeyDown(Keys.LeftAlt) || _currentKeyboardState.IsKeyDown(Keys.RightAlt);
         public bool IsKeyDown(Keys key) => _currentKeyboardState.IsKeyDown(key);
         public bool IsKeyUp(Keys key) => _currentKeyboardState.IsKeyUp(key);
         public Keys[] GetPressedKeys() => _currentKeyboardState.GetPressedKeys();
-        public void GetPressedKeys(Keys[] keys) => _currentKeyboardState.GetPressedKeys(keys);
```

Comment:
KeyboardState.GetPressedKeys(Keys[]) is an MG extension. It doesn't exist in FNA.

Reply:
Could this be moved to an extension method as well for compatibility with a note about this allocating a new array from the Array.Copy?

Reply:
Done.